packages/at_commons/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.0.53
+- fix: Modify "enroll" verb regex.
+- fix: Introduce "EnrollParams" class to encapsulate enrollment attributes.
```

Comment:
Both of the above were features not fixes - please use  'feat' not fix

Reply:
Sure Gary